explorations/ml_03_darts.py

```diff
@@ -0,0 +1,101 @@
+import pandas as pd
+
+from hrpe.data.load import (
+    filter_data_by_time,
+    load_minute_data,
+    load_hh_data,
+    load_maxmin_data,
+)
+from hrpe.features.deltas import calculate_deltas
+from hrpe.features.transform import minute_data_to_hh_data
+from hrpe.models.eval import score_model
+from hrpe.models.ets import ETS
+from hrpe.submit.create_submission import create_submissions_file
+
+from darts import TimeSeries
+from darts.utils.statistics import (
+    check_seasonality,
+    plot_acf,
+    plot_residuals_analysis,
+    plot_hist,
+)
+from darts.models import NaiveSeasonal, AutoARIMA, ExponentialSmoothing, Theta
+from darts.metrics import mape, mse
+
+data = load_minute_data("staplegrove")
+hh_data = minute_data_to_hh_data(data)
```

Comment:
Do we check this lines up with the HHdata? rather make a function that just joins / makes this

Reply:
we can source this from min-max and hh data in future yes